mobify/platform-scaffold
# Use walk to check the build folder for files larger than 2MB

Through discussions it was decided that the best way to verify that _minified_ build files are less than 2MB (to get gzipped on the CDN) is to examine the files in the `build` folder. 

## Changes
- Use [walk](https://www.npmjs.com/package/walk) to traverse the `build` folder and make sure each file is less than 2MB

## How to test-drive this PR
- Rebuild on CI https://circleci.com/gh/mobify/platform-scaffold/tree/verify-build-files-size and verify there is no change (all files are currently less than 2MB)
- To test the failure state locally, change the threshold on line 33 and 
```
npm run prod:build
node lighthouse/check-score.js
```
